### PR TITLE
Team Schedules

### PR DESCRIPTION
**Pull request by jmichaliga**
Creates a Team Page with Schedules

### CODE REVIEW

File: btsh-next/components/Users/Account.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  }, [session]);
+
+  async function getProfile() {
+    try {
+      setLoading(true);
+      const user = supabase.auth.user();
+
+      let { data, error, status } = await supabase
+        .from("profiles")
+        .select(`username, website, avatar_url`)
```
Comment (apathetic) [inline btsh-next/components/Users/Account.js:23]: aside: the template literal tripped me up until i learned you can perform queries in supabase this way.  would expanding it to separate lines to distinguish it / or something similar be useful?

File: btsh-next/components/Users/Avatar.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      )}
+      <div style={{ width: size }}>
+        <Label className="button primary block" htmlFor="single">
+          {uploading ? "Uploading ..." : "Upload"}
+        </Label>
+        <input
+          style={{
+            visibility: "hidden",
+            position: "absolute",
+          }}
```
Comment (apathetic) [inline btsh-next/components/Users/Avatar.js:76]: or maybe: 

```suggestion
          className="absolute invisible"
```

**Review by apathetic — APPROVED** [review]
dude, i cannot imagine the amount of work this was. it looks good; nice clean code, easy to follow.  nice work !

File: btsh-next/components/GameCard/GameCard.tsx [hunk 3]
```diff
@@ -1,17 +1,18 @@
-import { Box, Text } from "theme-ui";
+import Link from "next/link";
+import { Box, Heading, Text, Link as ThemeLink } from "theme-ui";
 import styles from "./GameCard.module.scss";
 
 interface GameCardProps {
   game: Game;
```
[the remark this answers is not in the document]
Comment (jmichaliga) [reply btsh-next/components/GameCard/GameCard.tsx:7]: Thanks for commenting! Game is in the global declared types in `global.d.ts`

File: btsh-next/components/Users/Avatar.js (hunk 2 above)
Comment (jmichaliga) [reply btsh-next/components/Users/Avatar.js:76]: Nice. Yeah, there was a little supabase boilerplate code.

**Pull request merged**